server/core/redis/redis_test.go

```diff
@@ -79,6 +79,88 @@ func TestRedisWithTLS(t *testing.T) {
 	_ = newTestRedisTLS(s)
 }
 
+func TestGetQueueByLock(t *testing.T) {
```

Comment:
There's a lot of repetition between this file and `boltdb_test.go`. Would it be possible to combine all these in one generic file, and let them run using both redis and boltdb (since we expect the exact same behavior)?

Reply:
Perhaps. Did you want to do this refactoring as a separate pr?

Reply:
I think this was answered by @jamengual in this comment: https://github.com/runatlantis/atlantis/pull/2782#issuecomment-1407742794

But if this is a valid option, I can try to work on it next.